src/commonMain/kotlin/software/momento/kotlin/sdk/responses/cache/control/CacheCreateResponse.kt

```diff
@@ -0,0 +1,23 @@
+package software.momento.kotlin.sdk.responses.cache.control
+
+import software.momento.kotlin.sdk.exceptions.SdkException
+
+/** Response for a create cache operation  */
+public interface CacheCreateResponse {
```

Comment:
I think using sealed classes for the responses instead of interfaces would be more idiomatic Kotlin.